pyquil/api/_quantum_computer.py

```diff
@@ -796,7 +797,11 @@ def get_qc(
     .. _QCS API Docs: https://docs.api.qcs.rigetti.com/#tag/endpoints
     """
 
-    client_configuration = client_configuration or QCSClient.load()
+    if client_configuration is None:
+        try:
+            client_configuration = QCSClient.load()
+        except LoadClientError:
+            client_configuration = QCSClient()
```

Comment:
I believe there is at least one issue with this, it won't respect env var overrides for `QCS_APPLICATIONS_(QUILC|QVM)_URL`, as the qcs-api-client doesn't check for those values in its default implementation. Assuming we could add support for that, is this a reasonable fallback?

Reply:
This seems reasonable to me though I think we should also log the exception so that it's clear what's happening